app/src/main/java/org/mozilla/fenix/addons/InstalledAddonDetailsFragment.kt

```diff
@@ -100,6 +102,8 @@ class InstalledAddonDetailsFragment : Fragment() {
                         runIfFragmentIsAttached {
                             this.addon = it
                             switch.isClickable = true
+                            privateBrowsingSwitch.isClickable = true
```

Comment:
We need to update the PB UI we the new add-on `privateBrowsingSwitch.isChecked = it.isAllowedInPrivateBrowsing()`. I will be better to add it before we make it visible this way is not going to blink
See https://github.com/mozilla-mobile/fenix/pull/10273#issuecomment-622408827

Reply:
> > If I `enable PB` -> disable the add-on -> go back to add-ons admin -> go to the details of the add-on -> enable the add-on -> **then PB is disable but it should be enable** see the link below:
> > https://drive.google.com/file/d/1PkFYqU9GCrAmz_4rcTEIObZz__3FYW2U/view?usp=sharing
> 
> I think the problem is that when we enter to the add-on details page with an disabled add-on, we don't have access to certain properties like (PB and optionsPageUrl), when we enable the add-on , get this properties back, I think we have to update the PB UI with the latest add-on. For this reason if you go back to the add-on admin page and re-enter it has the right state.

@Amejia481 I think this is related to https://bugzilla.mozilla.org/show_bug.cgi?id=1634493.
 When we enable/disable, the new web-extension/add-on instance we get back from GV does not persist previously set `allowedInPrivateBrowsing`.

Reply:
@psymoon when I added `privateBrowsingSwitch.isChecked = it.isAllowedInPrivateBrowsing()` the issue goes away, did you try it?